kuma/users/views.py

```diff
@@ -105,27 +105,32 @@ def profile_view(request, username):
     docs_feed_items = None
 
     context = {
-        'profile': profile,
         'demos': demos,
         'demos_paginator': demos_paginator,
         'demos_page': demos_page,
+        'detail_user': detail_user,
         'docs_feed_items': docs_feed_items,
     }
-    return render(request, 'users/profile.html', context)
```

Comment:
Here's the part that sets ``detail_user`` to the user who is being displayed.  I don't see anything in the code that would require swapping the viewing and the viewed user in the template - will check when I have live code and a database.